src/frontend/src/lib/components/dapps/DappModalDetails.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import DappModal from '$lib/components/dapps/DappModal.svelte';
+	import { modalDAppDetails } from '$lib/derived/modal.derived';
+	import { modalStore } from '$lib/stores/modal.store';
+	import type { OisyDappDescription } from '$lib/types/oisyDappDescription';
+
+	let selectedDapp: OisyDappDescription | undefined = undefined;
+	$: selectedDapp = $modalDAppDetails
+		? ($modalStore?.data as OisyDappDescription | undefined)
+		: undefined;
+</script>
+
+{#if nonNullish(selectedDapp)}
```

Comment:
You might want to also use a symbol here to identity this particular modal. Just in case it would be reused (some day).

Reply:
There is already a condition for `modalDAppDetails` to be true a few lines above which under the hood checks `$modalStore?.type === 'dapp-details'`. Or did I get your comment wrong?

Reply:
Indeed ;)

For example `SendButtonWithModal` - see the `const modalId = Symbol();`

```
<script lang="ts">
	import SendButton from '$lib/components/send/SendButton.svelte';
	import { modalStore } from '$lib/stores/modal.store';

	export let open: (modalId: symbol) => void;
	export let isOpen: boolean;

	const modalId = Symbol();
</script>

<SendButton on:click={() => open(modalId)} />

{#if isOpen && $modalStore?.data === modalId}
	<slot name="modal" />
{/if}
```

I just want to avoid to ends with two modals in the DOM in case somethine goes wrong.  But let's do that in another PR if it makes sense to you too.

Reply:
Ah, got it. Yeah, let's do it in a new PR.